test/sql/alter/add_pk/test_add_pk_rollback.test

```diff
@@ -25,7 +25,7 @@ INSERT INTO test VALUES (2, 1);
 statement error
 COMMIT
```

Comment:
Some more ideas for tests:

* Multiple (composite) unique indexes, with also gaps in the columns (i.e. primary key on col 3, unique key on col 5 or something like that)
* For the multiple unique indexes - also updates on a part of those indexes (i.e. you update part of a composite key, or only one of the keys in one of the unique indexes but not all keys)
* Generated columns before unique columns (i.e. `x GENERATED AS .., id INTEGER PRIMARY KEY`)
* Can we add a test with an index with expressions (e.g. `CREATE UNIQUE INDEX unique_index ON tbl(strlen(col1||col2));`
* Can we add a concurrent test that uses the deletes + inserts (i.e. using `concurrentloop` where all threads are doing these operations)
* Can we add a persistent storage test where we serialize the index while it has multiple rows in one of the leafs?